Documentation/policy/index.rst

```diff
@@ -15,7 +15,7 @@ This chapter documents the policy language used to configure network policies
 in Cilium. Security policies can be specified and imported via the following
 mechanisms:
 
-* Using Kubernetes `NetworkPolicy` `CiliumNetworkPolicy` and `CiliumClusterwideNetworkPolicy`
+* Using Kubernetes `NetworkPolicy`, `CiliumNetworkPolicy` and `CiliumClusterwideNetworkPolicy`
```

Comment:
Oh, didn't we rename this to `CiliumClusterNetworkPolicy`?

Reply:
No, @tgraf was okay with the name `CiliumClusterwideNetworkPolicy` since we already had a discussion around this in #sig-policy meeting where the name was changed from `CiliumGlobalNetworkPolicy` to `CiliumClusterwideNetworkPolicy`. I think we can have a discussion again in the next policy meeting just to have a common consensus around this?

Reply:
It's not a big deal, just wanted to make sure we're being consistent. If you ever wanted to change it though, now's the time---before we release ;)

Up to you, happy to either discuss in community / policy meeting or just drop it.